patterns/2-structured/trusted-committer.md

```diff
@@ -75,50 +73,49 @@ README.  As an example:
 
 ### Maintaining Trusted Committer Relationships
 
-When a new Trusted Committer is minted it’s a good idea to keep them in the
-loop as you continue to iterate on your project. This can be as simple as
-inviting them to your project channel or as involved as including them in your
+Once you formalize a new Trusted Committer, it is a good idea to keep them in the
+loop as you continue to iterate on your project. Keeping them in the loop can be as
+simple as inviting them to your project channel or as involved as including them in your
 planning sessions. More opportunities for involvement gives Trusted Committers
 a path to Maintainer if they so desire.
 
-Besides keeping Trusted Committers informed it’s a good idea to check in on a
-regular basis. A good cadence is every week, but as the Trusted Committer
-settles in this can drop to every few weeks or so. The purpose of these
-check-ins is to make sure the Trusted Committer feels supported in their new
-role, like a 1:1 with your manager. If things aren’t going well, listen and
-try to understand what is preventing the Trusted Committer from being successful.
-If things are going well, [thank the Trusted Committer for their continued
-effort][praise] in making the project successful and set a new date to check-in.
+Besides keeping Trusted Committers informed, it is good to check in on a
+regular basis. A suggested cadence is to start with every week before gradually
+progressing to every few weeks. The purpose of these check-ins is to make sure the
+Trusted Committer feels supported in their new role. Analogous to a 1:1 with your
+manager, if there are any issues, listen and empathize to try and understand
+what is preventing the Trusted Committer from being successful. Always
+[thank the Trusted Committer for their continued effort][praise] in making the project successful and set a new date to check-in.
 
 ### Sunsetting a Trusted Committer
 
-There comes a time when removal of a Trusted Committer is necessary, for
-example:
+There are times which necessitate removing a Trusted Committer, such as if the Trusted
+Committer:
```

Comment:
```suggestion
Committer is:
```

If I change it like this, I would be able to read it as a full sentence.
Is that better?